src/types/mod.rs

```diff
@@ -31,8 +31,9 @@ pub use self::memoryview::PyMemoryView;
 pub use self::module::{PyModule, PyModuleMethods};
 pub use self::none::PyNone;
 pub use self::notimplemented::PyNotImplemented;
-pub use self::num::PyLong;
-pub use self::num::PyLong as PyInt;
+pub use self::num::PyInt;
+#[deprecated(since = "0.23.0", note = "use `PyInt` instead")]
+pub use self::num::PyInt as PyLong;
```

Comment:
From a quick test, this does not seem to actually show any deprecation warning if the `PyLong` name is used. So we probably want to use a type alias to archive that.

```suggestion
#[deprecated(since = "0.23.0", note = "use `PyInt` instead")]
pub type PyLong = self::num::PyInt;
```

I _think_ this should not have any breaking side effects.

Reply:
Thank you for the review, appreciate your time!
I've implemented your suggestion.
I didn't put the type alias in `mod.rs`, but next to the `PyInt` struct definition instead.

Reply:
That's fine with me 👍